spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/repository/JdbcDataflowTaskExecutionDaoTests.java

```diff
@@ -56,18 +53,14 @@ public class JdbcDataflowTaskExecutionDaoTests {
 
 	@Autowired
 	private TaskRepository taskRepository;
-	@Autowired
-	private AggregateExecutionSupport aggregateExecutionSupport;
-	@Autowired
-	private TaskDefinitionReader taskDefinitionReader;
 
 	@Test
 	@DirtiesContext
 	public void testGetTaskExecutionIdsByTaskName() {
 		String taskName = UUID.randomUUID().toString();
-		List<AggregateTaskExecution> taskExecutions = createSampleTaskExecutions(taskName, 4);
-		for (AggregateTaskExecution taskExecution : taskExecutions) {
-			taskRepository.createTaskExecution(taskExecution.toTaskExecution());
+		List<TaskExecution> taskExecutions = createSampleTaskExecutions(taskName, 4);
+		for (TaskExecution taskExecution : taskExecutions) {
+			taskRepository.createTaskExecution(taskExecution);
 		}
```

Comment:
```suggestion
		createSampleTaskExecutions(taskName, 4).forEach(taskRepository::createTaskExecution);
```

This is good candidate for consolidating a bit, wdyt?